source/js/main.js

```diff
@@ -275,7 +278,7 @@ let main = {
       apps.push(
         new Promise(resolve => {
           ReactDOM.render(
-            <JoinUs {...props} whenLoaded={() => resolve()} />,
+            <I18nProvider language={getCurrentLanguage()} catalogs={catalogs}><JoinUs {...props} whenLoaded={() => resolve()} /></I18nProvider>,
```

Comment:
It might make sense for us to create a wrapper that does this part automatically, to replace direct calls to `ReactDOM.render`